src/FNA3D_Tracing.c

```diff
@@ -1549,6 +1549,12 @@ void FNA3D_Trace_SetStringMarker(const char *text)
 	SDL_UnlockMutex(traceLock);
 }
 
+void FNA3D_Trace_SetTextureName(void *texture, const char *text)
+{
+	// FIXME: Not implemented -kg
```

Comment:
```suggestion
	SDL_assert(0 && "Trace_SetTextureName not implemented -kg");
```

Reply:
Won't this break tracing though? Vs omit it from traces

Reply:
It'll warn when recording the trace, but it shouldn't affect the output data.

Reply:
In release builds it appears to do nothing, which is good enough for me